### DISCUSSION TITLE
Hardware Requirements “Production Deployments”

### DISCUSSION THREAD

**Discussion by mahmoudelhaj** [Security-Onion-Solutions/securityonion#9671] [Unsupported Versions]
I will install Distributed Security Onion, what are the minimum requirements ??
Are the following requirements sufficient? Specifically, disk space ??

**Manager node with separate search nodes**
8 CPU Cores, 16 GB RAM & **100 GB disk space.**

**Search Node**
4-8 CPU Cores, 16 GB RAM & **200 GB disk space.**

**Forward Node** 
4 CPU Cores, 16 GB RAM & **100 GB disk space.**

**Comment by cm-ops** [comment]
Have you had a chance to read through https://docs.securityonion.net/en/latest/hardware.html? Specifically the section on Manager with separate search node and sensor sections.

**Reply by mahmoudelhaj** [reply]
Yes, but I can't put more than 100 GB space manage node, any advice ?

**Reply by cm-ops** [reply]
You might be able to make this work, but we don't recommend it. Minimum specs are there for a reason.

**Reply by mahmoudelhaj** [reply]
thanks for help
